packages/bierzo-wallet/src/components/PageMenu/index.stories.tsx

```diff
@@ -57,6 +58,8 @@ const faultTx: ProcessedSendTransaction = {
     sender: "tiov1dcg3fat5zrvw00xezzjk3jgedm7pg70y222af3" as Address,
     recipient: "tiov1k898u78hgs36uqw68dg7va5nfkgstu5z0fhz3f" as Address,
   },
+  incoming: false,
+  outgoing: true,
 };
```

Comment:
What about stories where `incoming` and `outgoing` are both true here as well?

Reply:
Added in https://github.com/iov-one/ponferrada/pull/503/commits/0f6e665e8955732173197fe7bbe8357a3bb31caf. There is no storybook update since the transactions in the menu are only rendered when clicking the menu. But it is good to see it when testing manually.